manifests/ops-files/iaas/gcp/windows/ephemeral-disk.yml

```diff
@@ -0,0 +1,5 @@
+---
+- type: replace
+  path: /instance_groups/name=windows-worker/vm_extensions?/-
+  value: ((windows_ephemeral_disk_type))
```

Comment:
if disk_size too small --> refuses to make it a windows vm (maybe we will need this)